pages/checkout/index.js

```diff
@@ -533,6 +530,10 @@ class CheckoutPage extends Component {
 }
 
 CheckoutPage.propTypes = {
+  orderReceipt: PropTypes.oneOfType([
+    PropTypes.object,
+    PropTypes.oneOf([null]),
+  ]),
```

Comment:
```suggestion
  orderReceipt: PropTypes.oneOfType([
    PropTypes.object,
    null,
  ]),
```

Why would it be passed but be null?

Reply:
If you're asking why do we pass `null`, it's to know not to render the checkout/confirm page and also not to regenerate a checkoutToken here https://github.com/chec/commercejs-nextjs-demo-store/pull/90/files#diff-295c5e85a1a560e6f37f2ae1d68ab5e8R93.

Looking at it again, I'm not sure it's the best approach.

Reply:
Going to revert your suggestion as React's throwing this error now
`Warning: Invalid argument supplied to oneOfType. Expected an array of check functions, but received null at index 1.`